Source/Scene/ModelExperimental/MaterialPipelineStage.js

```diff
@@ -104,19 +104,13 @@ MaterialPipelineStage.process = function (
   // Configure back-face culling
   const model = renderResources.model;
   const cull = model.backFaceCulling && !material.doubleSided;
-  const translucent = defined(model.color) && model.color.alpha < 1.0;
-  renderResources.renderStateOptions.cull = {
-    enabled: cull && !translucent,
-  };
+  renderResources.renderStateOptions.cull.enabled = cull;
```

Comment:
The `translucent` check seems unsafe because there are multiple ways a models could become translucent. Better to set `cull` to true in the `AlphaPipelineStage` after all the stages have been true.